### PR TITLE
React/week1 homework

### PR DESCRIPTION
**Pull request by shpomp**
React/week1 homework: To do list

### CODE REVIEW

File: react/week1/homework/src/Button.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import PropTypes from "prop-types";
+// more action to come
+const Button = ({ color, text }) => {
+  return (
+    <button className="btn" style={{ backgroundColor: color }}>
```
Comment (senner008) [inline react/week1/homework/src/Button.js:5]: I suggest sticking to css sheet and not using inline styling

File: react/week1/homework/src/Header.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import PropTypes from "prop-types";
+import Button from "./Button";
+
+// const Header = (props) => {
+//   return <header>{props.title}</header>;
+// };
+
+const Header = ({ title }) => {
```
Comment (senner008) [inline react/week1/homework/src/Header.js:8]: Try to find more descriptive names than Header, Button, container etc.

**Review by senner008 — APPROVED** [review]
Good

A few details.
Remember that naming and separation goes together. Names are just as important
If you feel you need more challenge you can implement a fetch to an api with the useEffect Hook.

File: react/week1/homework/src/Button.js (hunk 1 above)
Comment (shpomp) [reply react/week1/homework/src/Button.js:5]: It was just to try props for styling 😊

File: react/week1/homework/src/Header.js (hunk 2 above)
Comment (shpomp) [reply react/week1/homework/src/Header.js:8]: > Good
> 
> A few details.
> Remember that naming and separation goes together. Names are just as important
> If you feel you need more challenge you can implement a fetch to an api with the useEffect Hook.

Thank you!

**Pull request merged**